diff -uNr python-emotion.old/configure.ac python-emotion/configure.ac
--- python-emotion.old/configure.ac	2011-06-02 17:12:34.000000000 +0000
+++ python-emotion/configure.ac	2011-08-25 17:05:13.450000105 +0000
@@ -63,7 +63,7 @@
 AC_SUBST(pkgconfig_requires_private)
 
 python_evas_version="0.7.3"
-pkgconfig_emotion_requires_libs="emotion >= 0.2.0.52190 evas >= 1.0.0 python-evas >= ${python_evas_version}"
+pkgconfig_emotion_requires_libs="emotion >= 0.2.0 evas >= 1.0.0 python-evas >= ${python_evas_version}"
 AC_SUBST(pkgconfig_emotion_requires_libs)
 PKG_CHECK_MODULES([EMOTION], [$pkgconfig_emotion_requires_libs])
 CYTHON_EMOTION_INCLUDEDIR=`$PKG_CONFIG --variable=includedir "python-evas >= ${python_evas_version}"`
